.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,52 @@
+name: Testing
+on:
+  push:
+    branches: ["*"]
+    tags-ignore: ["*"]
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  test:
+    name: Tests and style checks
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip setuptools twine wheel
+          pip install -r requires/development.txt
+      - name: Lint
+        run: |
+          flake8 sprockets/mixins/sentry tests.py setup.py docs/conf.py
+      - name: Verify distribution build
+        run: |
+          rm -fr dist
+          python setup.py bdist_wheel
+          python setup.py sdist
+          twine check --strict dist/*
```

Comment:
You can use `python -mbuild --sdist --wheel` here as well and jettison the requirement on the `wheel` package

Reply:
Makes sense, updated in ed7c70518d1a18527cbca257717a10c88608dbd7